### ISSUE TITLE
[Feature] Optional telemetry and extra features

### ISSUE THREAD

**Issue by JediMaster25** [stashapp/stash#3638]
By enabling telemetry, users would be able to see recommendations for scenes and performers based on their viewing history. This would make it easier to discover new content that is tailored to your specific interests. Also be able to see aggregated metadata like average rating, number of views, etc. Collecting and using the telemetry would also need to be implemented in stash-box.

**Comment by JediMaster25** [comment]
> Unless you mean recommending content that does not exist in your local Stash instance?

Exactly, the recommendations would be for scenes in the stash-box instance.

**Comment by DogmaDragon** [comment]
Core request is covered by existing issue.
